metabase/metabase
# Fix action editor UI issues

Epic #27581

### Description

1. Updates form creator's empty state (no title, decreases padding to `3em`)
2. Generalizes how various editor's right side components (form creator, data reference, action settings) are styled to take full available space. Instead of wrapping every component with a new div with `width: 100%; height: 100%`, that is now moved into `ActionCreator.styled` > `ModalRight`

### How to verify

1. Go to `/model/:modelId/detail/actions`
2. Click "New action"
3. Ensure you can't see the "Action parameters" title on the right side
4. Add a parameter (e.g. type a query like `{{ param }}`)
5. Ensure you can now see the "Action parameters" title
6. Add a few more parameters, and try to change their order (use the drag handle next to every parameter)
7. Ensure the parameter list's parent component takes full height and nothing gets visually cut when moving things around
8. Click on the data reference (book icon) and action settings (gear icon) at the top of the editor modal, play around with them
9. Ensure both sections take full width and height and don't look weird
10. Ensure the issue #28455 fixes isn't back 

### Demo

<details>
<summary>Before</summary>

![actioncreatorheight_360](https://user-images.githubusercontent.com/17258145/220697343-5a7dab49-27c0-48e9-8b41-88e99c49adb8.gif)

</details>

<details>
<summary>After</summary>

![CleanShot 2023-02-22 at 16 45 39](https://user-images.githubusercontent.com/17258145/220697652-1923be56-3a81-4343-8664-94993f23fceb.gif)

</details>
